CODE_OF_CONDUCT.md

```diff
@@ -59,12 +59,28 @@ reported by contacting the project team at jashkenas@gmail.com. The project team
 will review and investigate all complaints, and will respond in a way that it deems
 appropriate to the circumstances. The project team is obligated to maintain
 confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
-
 Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
 members of the project's leadership.
 
+## Moderation
+
+Edits of another user's comment must be clearly marked with **edit**,
+the moderator's username, and a timestamp for each occurrence. The only acceptable
+reasons for editing another user's comment are:
+
+1. to edit out violations of this Code of Conduct. These edits must include a rationale.
```

Comment:
That's not how I read it. Both are listed as "examples" (pro and con). They cover very different ground.